### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by robd003** [jazzband/django-recurrence#287]
Is this project abandoned?

**Comment by lino** [comment]
This is a [jazzband](https://jazzband.co/) project. As I am using this library actively, I kinda try to maintain it, but as you know how things are, sometimes other things are more important or at least shout louder in your general direction.

**Comment by thibaudcolas** [comment]
@lino it seems you raised the release failure in #291 / https://github.com/jazzband/help/issues/355. Do you know if there’s been any movement on it since January?

As-is it seems the package is stuck without a way to publish releases to PyPI.

**Comment by sheepman4267** [comment]
I'm also actively using this in multiple projects, and would be happy to share the load of maintaining the project.

**Comment by lino** [comment]
Well. The whole issue of ical and recurrence is very, very complex and I would really appreciate if the people who actively want to maintain libraries would concentrate their resources in a single project. But that's just my opinion.

**Comment by boosh** [comment]
Yeah I'd far rather that my lib became the v2 of django-recurrence. It solves a lot of outstanding issues (#116, #150, #155, etc) by supporting datetimes and provides several enhancements. However, it depends what any potential maintainers, etc. want. 

Then again, this repo has < 500 stars and ~125k monthly downloads and an apparently broken release pipeline, so it may not be worth the hassle for people to upgrade...

**Comment by lino** [comment]
Completely apart from the whole deployment issues thing, I would love to start planning for a v2 release, because it will allow us to remove some really janky behaviour of this library, but as breaking changes should not be unannounced, it needs some planning. I would really love to talk to you @boosh about what could be done for a v2.

**Comment by boosh** [comment]
Sure. I've created a discord server: https://discord.com/invite/EWTN65nN9f
